src/screens/SendSheet.js

```diff
@@ -464,7 +465,31 @@ export default function SendSheet(props) {
         );
 
         if (!lessThan(updatedGasLimit, gasLimit)) {
-          updateTxFee(updatedGasLimit, null, currentNetwork);
+          if (network === networkTypes.optimism) {
```

Comment:
can we create a separate function for this if else statement and reuse below in like 777

Reply:
done